addons/glusterfs/docker/entrypoint.py

```diff
@@ -0,0 +1,18 @@
+import json
+import os
+import subprocess
+
+obj = json.load(open("/etc/fluid/config.json"))
+
+mount_point = obj["mounts"][0]["mountPoint"]
+target_path = obj["targetPath"]
+
+os.makedirs(target_path, exist_ok=True)
+args = ["mount", "-t", "glusterfs", mount_point, target_path]
```

Comment:
By default, it's readonly unless accessMode is set.

Reply:
Default accessMode is readWrite unless set --read-only.
https://manpages.ubuntu.com/manpages/trusty/en/man8/glusterfs.8.html

Reply:
But it's readonly by default in dataset. It's not PVC.